docs/typeclasses/decideable.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: docs
+title: "Decidable"
+section: "typeclasses"
+source: "core/src/main/scala/cats/Decidable.scala"
+scaladoc: "#cats.Decidable"
+---
```

Comment:
Yay docs! You can nix the header and also make sure to add the entry to the `directory.conf`.

Reply:
Cool, thanks, I'll update this, there's probably a lot of stuff like this where convention has changed.